docs/topics/user_permissions.md

```diff
@@ -0,0 +1,51 @@
+# Users and permissions
+
+
+## What user permissions are defined in IoTP?
+
+
```

Comment:
Maybe that diagram already exist in componen-specific documentation and it is a matter of citing it.